pkg/entrypoint/sync/sync.go

```diff
@@ -71,6 +120,11 @@ func start(cfg *cmdConfig) error {
 		return err
 	}
 	log.Printf("running sync for plugin %s", cs.Config.PluginVersion)
+	syncInfoGauge.With(prometheus.Labels{
+		"plugin_version": cs.Config.PluginVersion,
+		"image":          cs.Config.Images.Sync,
+		"period_seconds": fmt.Sprintf("%d", int(cfg.interval.Seconds())),
```

Comment:
Question: is `period_seconds` always going to be different? if so, i'd strongly advise against using it as a label (as per https://prometheus.io/docs/practices/instrumentation/#do-not-overuse-labels ) and instead add another metric for it.

Reply:
This is an info gauge and the value is being read out of the config, so I read this as only changing if the sync pod is recreated with a new config (very infrequently).

Reply:
@thekad the `period_seconds` will change the least often from all labels in `syncInfoGauge`